applications/asset_tracker_v2/src/cloud/cloud_codec/cloud_codec.h

```diff
@@ -224,6 +224,23 @@ struct cloud_data_wifi_access_points {
 };
 #endif
 
+struct cloud_data_cloud_location {
+	/** Neighbor cell information is valid. */
+	bool neighbor_cells_valid;
+	/** Neighbor cells. */
+	struct cloud_data_neighbor_cells neighbor_cells;
+#if defined(CONFIG_LOCATION_METHOD_WIFI)
+	/** Wi-Fi access point information is valid. */
+	bool wifi_access_points_valid;
+	/** Wi-Fi access points. */
+	struct cloud_data_wifi_access_points wifi_access_points;
+#endif
+	/** Cloud location info timestamp. UNIX milliseconds. */
+	int64_t ts;
+	/** Flag signifying that the data entry is to be encoded. */
+	bool queued : 1;
+};
```

Comment:
Reordering the members of the struct might shave off a few bytes of required storage, which is always nice. Might get messy with ifdeffing, though

Reply:
Should we make it more messy with ifdefs or not? @simensrostad

Reply:
Well, if the tradeoff is worth it sure. If I'd keep ifdeffing to a minimum.

Reply:
16 bytes can be saved in ATv2 compilation with Wi-Fi enabled (without Wi-Fi obviously less) with this code:
```
struct cloud_data_cloud_location {
	/** Neighbor cells. */
	struct cloud_data_neighbor_cells neighbor_cells;
#if defined(CONFIG_LOCATION_METHOD_WIFI)
	/** Wi-Fi access points. */
	struct cloud_data_wifi_access_points wifi_access_points;
#endif
	/** Cloud location info timestamp. UNIX milliseconds. */
	int64_t ts;
	/** Neighbor cell information is valid. */
	bool neighbor_cells_valid;
#if defined(CONFIG_LOCATION_METHOD_WIFI)
	/** Wi-Fi access point information is valid. */
	bool wifi_access_points_valid;
#endif
	/** Flag signifying that the data entry is to be encoded. */
	bool queued : 1;
};
```
You can decide whether to change or not.

Reply:
Nah, just leave it as is